packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.test.js

```diff
@@ -14,11 +14,34 @@
 
 import React from 'react';
 import { shallow } from 'enzyme';
-import { Dropdown, Icon, Tooltip } from 'antd';
-import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { Dropdown, Icon } from 'antd';
+
+import CopyIcon from '../../../common/CopyIcon';
 
 import KeyValuesTable, { LinkValue } from './KeyValuesTable';
 
+describe('LinkValue', () => {
+  const title = 'titleValue';
+  const href = 'hrefValue';
+  const childrenText = 'childrenTextValue';
+  const wrapper = shallow(
+    <LinkValue href={href} title={title}>
+      {childrenText}
+    </LinkValue>
+  );
+
+  it('renders as expected', () => {
+    expect(wrapper.find('a').prop('href')).toBe(href);
+    expect(wrapper.find('a').prop('title')).toBe(title);
+    expect(wrapper.find('a').text()).toMatch(/childrenText/);
```

Comment:
Nit: I would typically recommend saving to a const, `const anchor = wrapper.find(...)`